### DISCUSSION TITLE
How to set "isValid" class if validation has completed and is valid?

### DISCUSSION THREAD

**Discussion by simplenotezy** [react-hook-form/react-hook-form#5152] [Q&A]
I'd like to add a class to my input if validation was successful. I'm aware that I can check for the `error` object but that will always be undefined, even before the user has touched the input.

I'd like to see if user touched the input, and the input is valid thereafter.

I could do it with useState, but I was wondering if there is an built-in approach

**Comment by bluebill1049** [comment]
share a codesandbox
